priv/repo/migrations/20201110152910_create_notifications.exs

```diff
@@ -0,0 +1,34 @@
+defmodule Skate.Repo.Migrations.StoreNotifications do
+  use Ecto.Migration
+
+  def up do
+    create table(:notifications) do
+      add(:created_at, :bigint, null: false)
+      add(:reason, :string, null: false)
```

Comment:
Any particular reason not to implement this as an `enum` type in Postgres?

Reply:
No, that makes sense.